airflow/cli/commands/provider_command.py

```diff
@@ -134,7 +134,7 @@ def connection_form_widget_list(args):
 def connection_field_behaviours(args):
     """Lists field behaviours."""
     AirflowConsole().print_as(
-        data=list(ProvidersManager().field_behaviours.keys()),
+        data=list(ProvidersManager().field_behaviours),
```

Comment:
There’s really no particular reason `data` needs to be a sequence; it can just be a `Collection` and avoid the `list` cast. This should be a follow-up PR instead though.